core/src/main/java/de/danoeh/antennapod/core/feed/util/ImageResourceUtils.java

```diff
@@ -13,9 +13,17 @@
     private ImageResourceUtils() {
     }
 
+    /**
+     * calls {@link ImageResourceUtils#getImageLocation(ImageResource, boolean)} without requesting a feed cover.
+     */
     public static String getImageLocation(ImageResource resource) {
+        return getImageLocation(resource, false);
+    }
+
+
+    public static String getImageLocation(ImageResource resource, boolean preferredFeedCover) {
```

Comment:
I think it might be more clear to use two different methods instead of one with parameters. Maybe `getEpisodeImageLocation` and `getEpisodeListImageLocation`?